storage/innobase/xtrabackup/test/t/all_files.sh

```diff
@@ -29,12 +29,6 @@ diff -u <( ( ( cd $dir1; find . | grep -Ev $ign_list )
 ./xtrabackup_tablespaces
 EOF
 
-if is_xtradb ; then
-    XTRA_DOUBLEWRITE="./xb_doublewrite"
-else
-    XTRA_DOUBLEWRITE=""
-fi
-
```

Comment:
we need this isn't it? if PXB is run against PS 8.0.19 or lower ?

Reply:
I have bumped the testing version to PS 8.0.20.

Reply:
but I think the test case can be changed to handle both. How else would we test PXB against lower versions and its compatibility?